metadata.yaml

```diff
@@ -33,6 +33,8 @@ requires:
     interface: kube-control
   aws:
     interface: aws-integration
+  coredns:
+    interface: coredns
```

Comment:
I don't think we should call this new interface 'coredns'. That's an implementation-specific name. We need a generic interface name so that when a new dns provider comes along, we can plug that in here too, and still have it make sense without changing the interface name.

I think 'kube-dns' would be a good name. That name is already taken, but the interface itself is deprecated so I think we can safely reuse the name.

Reply:
Per the team discussion, let's go with 'cluster-dns' - thanks!